assets/queries/k8s/kubelet_read_only_port_is_not_set_to_zero/query.rego

```diff
@@ -0,0 +1,25 @@
+package Cx
+
+import data.generic.k8s as k8sLib
+
+CxPolicy[result] {
+	resource := input.document[i]
+	specInfo := k8sLib.getSpecInfo(resource)
+	container := specInfo.spec.containers[j]
+
+	inArray(container.command, "kubelet")
+	containerArgs := object.get(container, "args", {})
```

Comment:
Please, use the function `get_nested_values_info` in line 11. I left a comment in #4940.

Additionally, I think `--read-only-port `can be defined inside `container.command`.